components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/resources/userstore.yaml

```diff
@@ -315,6 +315,42 @@ paths:
           $ref: '#/components/responses/ServerError'
         '501':
           $ref: '#/components/responses/NotImplemented'
+  '/userstores/meta/attributes/{type-id}':
```

Comment:
its not clear whats meant by `meta`.  shall we use a more clear word

Reply:
if it's meant to be metadata let's use the word metadata.

also i think `/userstores/{type-id}/metadata/attributes` would have a better resource representation

Reply:
meta is an already defined word. So changing it will not be okay. Should I create metadata word just for this?